scanner/keppel/go.mod

```diff
@@ -1,3 +1,35 @@
 module github.com/cloudoperators/heureka/scanners/keppel
```

Comment:
Is there a policy when to use github.com and when to use github.wdf.sap.com (like here: https://github.com/cloudoperators/heureka/blob/scanner_nvd/scanner/nvd/go.mod#L1) ?